docs/source/getting_started/benchmarks.rst

```diff
@@ -41,121 +37,160 @@ See the `benchmarking scripts <https://github.com/lightly-ai/lightly/tree/master
   "SwAV", "Res50", "256", "100", "67.2", "88.1", "75.4", "92.7", "49.5", "78.6", "`link <https://tensorboard.dev/experiment/Ipx4Oxl5Qkqm5Sl5kWyKKg>`_", "`link <https://lightly-ssl-checkpoints.s3.amazonaws.com/imagenet_resnet50_swav_2023-05-25_08-29-14/pretrain/version_0/checkpoints/epoch%3D99-step%3D500400.ckpt>`_"
   "VICReg", "Res50", "256", "100", "63.0", "85.4", "73.7", "91.9", "46.3", "75.2", "`link <https://tensorboard.dev/experiment/qH5uywJbTJSzgCEfxc7yUw>`_", "`link <https://lightly-ssl-checkpoints.s3.amazonaws.com/imagenet_resnet50_vicreg_2023-09-11_10-53-08/pretrain/version_0/checkpoints/epoch%3D99-step%3D500400.ckpt>`_"
 
-*\*We use square root learning rate scaling instead of linear scaling as it yields better results for smaller batch sizes. See Appendix B.1 in SimCLR paper.*
+*\*We use square root learning rate scaling instead of linear scaling as it yields better results for smaller batch sizes. See Appendix B.1 in the SimCLR paper.*
+
+Found a missing model? Track the progress of our planned benchmarks on `GitHub <https://github.com/lightly-ai/lightly/issues/1197>`_.
```

Comment:
this sentence is weirdly placed here under ImageNet1k. Maybe add it to every dataset or put it at the top or bottom more centralized?

Reply:
The tracker is only about ImageNet1k so it has to stay in this section. Not sure why you think it's weirdly placed. What's the problem?

Reply:
Sorry, I was under the impression that its a general Q for model suggestions. Yes, then its at the correct place